pkgs/by-name/ce/centerpiece/package.nix

```diff
@@ -35,12 +36,18 @@ rustPlatform.buildRustPackage rec {
   postFixup = lib.optional stdenv.isLinux ''
     rpath=$(patchelf --print-rpath $out/bin/centerpiece)
     patchelf --set-rpath "$rpath:${
-      lib.makeLibraryPath [
+      lib.makeLibraryPath ([
         libGL
         libxkbcommon
         vulkan-loader
         wayland
       ]
+      ++ lib.optionals enableX11 (with xorg; [
+        libX11
+        libXcursor
+        libXi
+        libXrandr
+      ]))
```

Comment:
Why can we not add these to build inputs?

Reply:
Done